site/content/docs/5.3/getting-started/download.md

```diff
@@ -67,7 +67,7 @@ Pull in Boosted's **source files** into nearly any project with some of the most
 Install Boosted in your Node.js powered apps with [the npm package](https://www.npmjs.com/package/boosted):
 
 ```sh
-npm install boosted@v5.3.0-alpha1
+npm install boosted@v5.3.0-alpha2
```

Comment:
We missed gathering a commit at some point from Bootstrap, upstream it's using the config version. I'll do it right after this PR.

Reply:
Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1926